src/renate/defaults.py

```diff
@@ -57,6 +57,7 @@
 JOB_NAME = "renate"
 SUPPORTED_TUNING_MODE = ["min", "max"]
 SUPPORTED_TUNING_MODE_TYPE = Literal["min", "max"]
+RETAIN_INTERMEDIATE_STATE = True
```

Comment:
Let's set it to `False`.

Reply:
The current mode of operation is `True`. We shouldn't alter that, right?

Reply:
given that we never used the intermediate states, I'm fine changing the default behavior